src/components/courses/AdminPage.tsx

```
@@ -0,0 +1,411 @@
+    }
+  }, [reloadData]);
+
+  useEffect(() => {
+    fetchCourses();
+  }, []);
+
+  const handleAddCourse = (formData: FormData) => {
+    try {
+      let user: any;
+      let userJson = localStorage.getItem("user");
+      if (userJson) user = JSON.parse(userJson);
```

Biome

[error] This let declares a variable that is only assigned once.

'userJson' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)